lib/games_manager.rb

```diff
@@ -173,91 +173,69 @@ def get_season_games(season)
   end
 
   def total_home_goals
-    home_team_id_goals = {}
-    games.each do |game|
-      if home_team_id_goals[game.home_team_id].nil?
-        home_team_id_goals[game.home_team_id] = game.home_goals
-      else
-        home_team_id_goals[game.home_team_id] += game.home_goals
-      end
+    team_id_by_home_goals = games.map do |game|
+      [game.home_team_id, game.home_goals]
     end
-    home_team_id_goals
+    sum_values(team_id_by_home_goals)
   end
-
-  def total_home_games
-    home_team_id_games = {}
-    games.each do |game|
-      if home_team_id_games[game.home_team_id].nil?
-        home_team_id_games[game.home_team_id] = 1
-      else
-        home_team_id_games[game.home_team_id] += 1
-      end
+  
+  def total_away_goals
```

Comment:
It looks like Aidan's nice 'map' refactors are from the prior (also was the first) refactor merge.